pkg/route/api/v1/types.go

```diff
@@ -41,7 +41,11 @@ type RouteSpec struct {
 
 	// To is an object the route points to. Only the Service kind is allowed, and it will
 	// be defaulted to Service.
-	To kapi.ObjectReference `json:"to"`
+	To RouteTargetReference `json:"to"`
+
+	// AdditionalTos is an extension of the 'to' field. If more than one service needs to be
+	// pointed to, then use this field
+	AdditionalTos []RouteTargetReference `json:"additionalTos,omitempty"`
```

Comment:
@liggitt opinions please on naming of this field.  We have a route, which points to one object by default.  We want to set up additional / alternative / extra objects and spreads across them.  We may in the future have behavior that swaps / replaced the primary and secondaries.  We can't change "to" due to back compat.  I'm leaning towards calling this field "alternates" but want to get a second opinion.